game.go

```diff
@@ -115,13 +115,15 @@ func nextGameState(state GameState) GameState {
 
 type Game struct {
 	GameState
-	ID           string    `json:"id"`
-	CreatedAt    time.Time `json:"created_at"`
-	UpdatedAt    time.Time `json:"updated_at"`
-	StartingTeam Team      `json:"starting_team"`
-	WinningTeam  *Team     `json:"winning_team,omitempty"`
-	Words        []string  `json:"words"`
-	Layout       []Team    `json:"layout"`
+	ID            string    `json:"id"`
+	CreatedAt     time.Time `json:"created_at"`
+	UpdatedAt     time.Time `json:"updated_at"`
+	StartingTeam  Team      `json:"starting_team"`
+	WinningTeam   *Team     `json:"winning_team,omitempty"`
+	Words         []string  `json:"words"`
+	Layout        []Team    `json:"layout"`
+	TimerSettings []int32   `json:"timer_settings"`
```

Comment:
All we really care about here is the duration, yeah? The fact that the user enters the minutes and seconds fields separately isn't useful anymore, so IMO we should collapse this into a single, immediately understandable value. Since `time.Duration` has its own String method that would be annoying to parse on the frontend, how about just an integer milliseconds?

```suggestion
	TimerDurationMillis int64   `json:"timer_duration_millis,omitempty"`
```